external/zsh/_msfvenom

```diff
@@ -207,6 +207,20 @@ _msfvenom_platform() {
   _describe -t platforms 'available platforms' _msfvenom_platforms_list || compadd "$@"
 }
 
+_msfvenom_payload() {
+  local cacheFile="$HOME/.msf4/store/modules_metadata.json"
+  local -a _msfvenom_payloads_list
+  [ -f "$cacheFile" ] || cacheFile="/opt/metasploit/db/modules_metadata_base.json"
+  [ -f "$cacheFile" ] || cacheFile="./db/modules_metadata_base.json"
+  if [ ! -f "$cacheFile" ]; then
+    _message -r "Cannot find metasploit cache file. Run msfconsole to populate it"
+    compadd "$@"
+  else
+    _msfvenom_payloads_list=("${(f)$(sed -n '/"type": "payload"/,/"ref_name"/p' "$cacheFile" | grep -E '(ref_name|description)' | cut -d '"' -f 4 | sed -n 'h;n;p;g;p' | sed 'N;s/\n/:/')}")
```

Comment:
Just confirming if new lines will cause any issues here?

I noticed some of the descriptions have new lines present:

> mainframe/shell_reverse_tcp:Listen for a connection and spawn a command shell.\n          This implementation does not include ebcdic character translation,\n          so a client with translation capabilities is required.  MSF handles\n          this automatically.

Reply:
It won't cause any issues although the output won't go onto multiple lines. The help message will just contain `\n`. Do you think it would be fair to simply remove everything except the first line?

Reply:
Removing everything but the first line would result in this:

![image](https://user-images.githubusercontent.com/1909202/77785795-95bd5b00-7054-11ea-9aba-125906e61af4.png)